scripts/insert_into_appointments.sql

```diff
@@ -1 +1,6 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments (patient_id, doctor_id, appointment_time, appointment_date) VALUES
+(8, 4, '10 AM', '2016-08-12'),
+(5, 6, '1 PM', '2016-08-17'),
+(8, 6, '10 AM', '2016-08-12'),
+(7, 4, '1 PM', '2016-08-17');
```

Comment:
Good but remember that you set up your appointment_time column to accept time values and not strings. So '10 AM' would have to be '1000' and '1 PM' would be '1300'.